### PR TITLE
Update missing Email Template List

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) updates the missing Email Template List for:
- Magento_CustomerBalance
- Magento_GiftCard
- Magento_GiftCardAccount
- Magento_GiftRegistry
- Magento_InventoryInStorePickupSales
- Magento_Invitation
- Magento_NegotiableQuote
- Magento_Reminder
- Magento_Reward
- Magento_Rma
- Magento_ScheduledImportExport

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/email-template-list.html#magento_customerbalance

### CODE REVIEW

File: src/marketing/email-template-list.md [hunk 1]
```diff
@@ -96,51 +96,50 @@ The list of email templates is organized alphabetically by Magento module.
...
+| Order is Ready for Pickup | **Page:** Sales > [Sales Emails]({% link configuration/sales/sales-emails.md %}) <br/>**Section:** Order Ready For Pickup in Store<br/>**Field:** Order Ready For Pickup Email Template |
+| Order is Ready for Pickup For Guest | **Page:** Sales > [Sales Emails]({% link configuration/sales/sales-emails.md %}) <br/>**Section:** Order Ready For Pickup in Store<br/>**Field:** Order Ready For Pickup Email Template for Guest|
 
 ## Magento_Invitation
 
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| Customer Invitation | **Page:** <br/>**Section:** <br/>**Field:** |
+| Customer Invitation | **Page:** Sales > [Sales Emails]({% link configuration/customers/invitations..md %})<br/>**Section:** Email<br/>**Field:** Customer Invitation Email Template |
```
Comment (leslietilling) [inline src/marketing/email-template-list.md:130]: I am not sure on the "Sales Emails" link that is going to the Customers invitations page here. Was this intentional?

Comment (EvheniiaKolkhiieva) [reply src/marketing/email-template-list.md]: Hi @leslietilling , thank you for review. Yes you are right :) My mistake. I have changed this to -` | Customer Invitation | **Page:** Customers > [Invitation]({% link configuration/customers/invitations.md %})<br/>**Section:** Email<br/>**Field:** Customer Invitation Email Template | `
Please approve☺️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**